src/gui/mzroll/tabledockwidget.cpp

```diff
@@ -1765,6 +1765,61 @@ void TableDockWidget::showIntegrationSettings()
   log.exec();
 }
 
+void TableDockWidget::moveSelectedRows(QString destinationTableName) {
+
+  QString sourceTableName = this->titlePeakTable->text();
+  QList<QPointer<TableDockWidget>> peaksTableList = _mainwindow->getPeakTableList();
+  QList<shared_ptr<PeakGroup>> selected = getSelectedGroups();
+  QString bookmarkTableName = _mainwindow->bookmarkedPeaks->titlePeakTable->text();
+
+  if(bookmarkTableName == destinationTableName) {
+    BookmarkTableDockWidget* destinationPeakTable = _mainwindow->bookmarkedPeaks;
+    for(auto group : selected) {
+      destinationPeakTable->addPeakGroup(group.get());
+    }
+    destinationPeakTable->showAllGroups();
+    destinationPeakTable->setVisible(true);
+  } else {
+    TableDockWidget* destinationPeakTable = nullptr;
+    for (auto table : peaksTableList) {
+        if (table->titlePeakTable->text() == destinationTableName) {
+            destinationPeakTable = table;
+        }
+        if(destinationPeakTable != nullptr) {
+          break;
+        }
```

Comment:
The `if` followed by a `break` can simply be a part of the `if` above it. That is, if `destinationPeakTable` is set to a table, then we break out of the loop:
```C++
if (table->titlePeakTable->text() == destinationTableName) {
  destinationPeakTable = table;
  break;
}
```

Reply:
oh yes, will do